### DISCUSSION TITLE
articles/failure-of-zhimaishu

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#129] [Comments]
https://hutusi.com/articles/failure-of-zhimaishu 

 背景

**Comment by hcen1997** [comment]
作为对于git架构的复习:
实体书作为BLOB数据,像git一样,可以不用'登记、借书、确认，需要约定时间地点取书'.
可以借助现在或者未来的共享存储空间(丰巢,快递暂存柜).
'只买书'只要做到准确追踪书籍的位置就可以了.

**Comment by hutusi** [comment]
> @hcen1997
> 作为对于git架构的复习:
> 实体书作为BLOB数据,像git一样,可以不用'登记、借书、确认，需要约定时间地点取书'.
> 可以借助现在或者未来的共享存储空间(丰巢,快递暂存柜).
> '只买书'只要做到准确追踪书籍的位置就可以了.
> 
> 

赞，很好的想法 👍

**Comment by petite-dd** [comment]
我也买了很多实体书，今年也买了几千块钱的。图书漂流是我会尝试的东西，可能就是需求小众了一些吧，想到就行动也是我的风格👍

**Comment by hutusi** [comment]
@petite-dd 确实太小众了，像 bookcrossing 这种20年前的老站，现在也不太活跃。

**Comment by linlinzzo** [comment]
想法挺好的。

**Comment by magic-moon** [comment]
结果是失败的，过程是成功的，就像是做了一次失败的实验，训练了实验的技能
